velox/type/tests/TimestampConversionTest.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include "velox/type/TimestampConversion.h"
+#include <common/base/tests/GTestUtils.h>
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
```

Comment:
Include `#include "velox/common/base/tests/GTestUtils.h"` we can remove `#include <gtest/gtest.h>`